CallPlayAudio/App.config

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<startup>
+		<supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/>
+	</startup>
+	<appSettings>
+		<!-- 1. Configurations related to Communication Service resource.-->
+
+		<!-- Connection string of Azure Communication Service Resource. -->
+		<add key="Connectionstring" value="%Connectionstring%"/>
+		<!-- Phone number provisioned for the ACS resource (in E.164 Format, e.g. +1425XXXYYYY). This is an alternative phone id -->
+		<add key="SourcePhone" value="%SourcePhone%"/>
+		<!--
+    Destination identities to call. 
+        Format: "Target1(PhoneNumber)"
+		For e.g. "+1425XXXAAAA"
+    -->
+		<add key="DestinationIdentities" value="%DestinationIdentities%"/>
+		<add key="MaxRetryCount" value="2"/>
+
+		<!-- 2. Configurations related to environment-->
+
+		<!-- Directory where ngrok.exe is saved. -->
+		<add key="NgrokExePath" value="%NgrokExePath%"/>
+		<!-- Secret for validating incoming request. -->
+		<add key="SecretPlaceholder" value="h3llowW0rld"/>
+
+		<!-- 3. Optional configurations for generating cusom audio message-->
+
+		<!-- Cognitive service key (Optional). -->
+		<add key="CognitiveServiceKey" value=""/>
+		<!-- Cognitive service region (Optional). -->
+		<add key="CognitiveServiceRegion" value=""/>
+		<!-- Custom message that will be translated by Azure Cognitive service (Optional). -->
+		<add key="CustomMessage" value="Hello, this is a reminder call. If you would like to speak with a representative Press 1 or 2 if you want to hang up."/>
```

Comment:
Do we need this congnitive Service configuration or use this in PlayAudio sample anywhere ?